pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [tool.black]
 line-length = 100
-target-version = ['py37', 'py38']
```

Comment:
We should add that comment here as well to include py39 whenever a next version of black is released.

Reply:
changed the comment